config-connector/tests/testcases/projects/kpt/project-services/required_fields_only.yaml

```diff
@@ -0,0 +1,14 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+project-id: $DEFAULT_PROJECT_ID
```

Comment:
Can we call it PROJECT_ID for consistency?

Reply:
Ack. As I explained the previous PRs, I'll improve the PROJECT_ID related env vars later.